packages/react-native-web/src/exports/StyleSheet/index.js

```diff
@@ -1,6 +1,11 @@
+// @flow
+
 import { canUseDOM } from 'fbjs/lib/ExecutionEnvironment';
 import StyleSheet from './StyleSheet';
 
+type Atom = number | boolean | Object | Array<?Atom>;
+export type StyleObj = Atom;
```

Comment:
This creates a cyclic dependency between various files in `StyleSheet`. The types should live in a separate file